src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,25 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo" key={post.id}>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {`${post.title}`}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {`${post.body}`}
+    </p>
+
+    <CommentList comments={post.comments} />
```

Comment:
it would be more correct to move this condition here so as not to render the component itself if the comments are empty
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/54936044/e7bff39f-574a-42f9-b57b-9b6041580d40)